rllib/algorithms/algorithm.py

```diff
@@ -3,16 +3,18 @@
 import copy
 from datetime import datetime
 import functools
-import gym
+import gymnasium as gym
```

Comment:
Note for everything below: All `import gym` (and related) have been replaced by `import gymnasium as gym`.